Source/Database.cpp

```diff
@@ -74,7 +74,7 @@ Game Database::getGameById(unsigned int id)
     return {id, name, path, exe};
 }
 
-Game Database::getGameByName(QString name)
+Game* Database::getGameByName(QString name)
```

Comment:
It was a lazy hack to compare it to nullptr. I meant to make a TODO to
find a better way to do it.

https://github.com/Proj-Ascension/Client/blob/dev/Source/Library.cpp#L249

(I know it's god awful, I didn't know how to check for existence better)
On 07/07/15 18:21, Benjamin Cheng wrote:
> In Source/Database.cpp
> <https://github.com/Proj-Ascension/Client/pull/34#discussion_r34065542>:
> 
>> @@ -74,7 +74,7 @@ Game Database::getGameById(unsigned int id)
>>      return {id, name, path, exe};
>>  }
>>  
>> -Game Database::getGameByName(QString name)
>> +Game* Database::getGameByName(QString name)
> 
> Wait, any reason for making this a pointer (other than causing memory
> leaks)?
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Proj-Ascension/Client/pull/34/files#r34065542>.
>